firestore/tests/unit/v1/testdata/set-st-merge-both.textproto

```diff
@@ -30,6 +30,7 @@ set: <
       >
       update_mask: <
         field_paths: "a"
+        field_paths: "b"
```

Comment:
We can't change these `.textproto` files arbitrarily:  they represent cross-language conformance tests.  See: https://github.com/googleapis/conformance-tests/blob/bbfcb1392335ca2bf043d11483676dde78ebaa09/firestore/v1/set-st-merge-both.json